### ISSUE TITLE
[DevHome] [Dev Drive] In the textboxes when there is an error the textbox related to the error should have a red line under the text area 

### ISSUE THREAD

**Issue by mathewleeza** [microsoft/devhome#497]
### Steps to reproduce the bug

currently win ui 3 does not have a validation style for textbox, so we'd need to create a custom one. Since this does not affect the shipping of the product and is not a priority. This window pops up when a user clicks the "Create a Dev Drive to optimize performance" checkbox and then selects the customize hyperlink button as shown below:
![customize button that shows when checkbox is selected](https://github.com/microsoft/devhome/assets/105318831/c5dd44ec-fb55-4d82-bdff-e9982db9153c)

Note: The Windows Settings app does not include this either which is using win UI 2, so I don't think this is something that is easy to do.

Basically in the screenshot below the arrows point to the default white lines under the textbox, The designer wants them to turn red when there is an error

![previous Create Dev Drive window](https://github.com/microsoft/devhome/assets/105318831/a18d3d6a-6fbd-40d5-a661-0ef61b733f1e)

**Comment by betanzos** [comment]
Could you update the screenshot reference?

**Comment by bbonaby** [comment]
@betanzos  updated the screenshot reference. Thanks
